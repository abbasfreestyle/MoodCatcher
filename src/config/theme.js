const palette = {
  charcoal: '#393E41',
  trout: '#4A4C58',
  darkGreen: '#44BBA4',
  calmBlue: '#81DBD1',
  brightGreen: '#F2FCFB',
  grey: '#EEEEEE',
  white: '#FFFFFF'
};

const theme = {
  primary: palette.darkGreen,
  primaryVariant: palette.calmBlue,
  secondary: palette.brightGreen,
  surface: palette.white,

  onPrimary: palette.white,
  onPrimaryVariant: palette.white,
  onSecondary: palette.charcoal,
  onSurface: palette.trout,

  shadow: palette.darkGreen,
  disabled: palette.grey
};

export default theme;
