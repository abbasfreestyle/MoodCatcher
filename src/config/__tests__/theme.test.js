import theme from '../theme';

describe('Theme.', () => {
  describe('Snapshots.', () => {
    test('should match', () => {
      expect(theme).toMatchSnapshot();
    });
  });
});
