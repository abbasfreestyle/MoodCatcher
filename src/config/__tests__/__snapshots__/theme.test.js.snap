// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme. Snapshots. should match 1`] = `
Object {
  "disabled": "#EEEEEE",
  "onPrimary": "#FFFFFF",
  "onPrimaryVariant": "#FFFFFF",
  "onSecondary": "#393E41",
  "onSurface": "#4A4C58",
  "primary": "#44BBA4",
  "primaryVariant": "#81DBD1",
  "secondary": "#F2FCFB",
  "shadow": "#44BBA4",
  "surface": "#FFFFFF",
}
`;
